src/pages/settings.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from "react";
+import PageTemplate from "@/components/templates/PageTemplate";
+import SettingsRead from "@/components/organisms/SettingsRead";
+import SettingsChange from "@/components/organisms/SettingsChange";
+import SettingsDropDown from "@/components/organisms/SettingsDropDown";
+import api from "../lib/axios_settings";
+
+export default function SettingsIndex() {
+  const [settingsView, setSettingsView] =
+    useState<string>("drop down settings");
+  const [userData, setUserData] = useState<{}>({});
+  const [deviceData, setDeviceData] = useState<[]>([]);
+  const [selectedDeviceName, setSelectedDeviceName] =
+    useState<string>("hello ondo");
+  const [selectedDeviceIndex, setSelectedDeviceIndex] = useState<number>(-1);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+
+  console.log(selectedDeviceName);
+  console.log("SETTINGS", selectedDeviceIndex);
```

Comment:
Please remove console.log.

Reply:
I'm saving these for a feature I'm working on still  (dynamically updating the device index on the Settings Read page).  But I will delete them once that feature is complete!